bridge/engine/RecordingAudioForwarderSendJob.cpp

```diff
@@ -53,11 +60,25 @@ void RecordingAudioForwarderSendJob::run()
         {
             if (!packetCache->add(*_packet, nextSequenceNumber))
             {
-                logger::info("Dropping rec audio packet - cache issues...", "RecordingAudioForwarderSendJob");
-                return;
+                logger::warn("Failed to cache rec audio packet", "RecordingAudioForwarderSendJob");
```

Comment:
let the audio packet through also when packet cache is not accepting it.

Reply:
Yes, that make sense 👍